src/vector_of_array.jl

```diff
@@ -72,11 +72,7 @@ Base.@propagate_inbounds function Base.getindex(A::AbstractDiffEqArray{T, N},sym
   end
 
   if i === nothing
-    if issymbollike(i) && A.indepsym !== nothing && Symbol(i) == A.indepsym
-      A.t
```

Comment:
did you try the independent variable and see what it does?

Reply:
I think that case is covered now. Added tests too.